PostFreeSurfer/PostFreeSurferPipeline.sh

```diff
@@ -63,12 +63,15 @@ opts_AddMandatory '--lowresmesh' 'LowResMeshes' 'number' "usually '32', the stan
 opts_AddMandatory '--subcortgraylabels' 'SubcorticalGrayLabels' 'file' "location of FreeSurferSubcorticalLabelTableLut.txt"
 opts_AddMandatory '--freesurferlabels' 'FreeSurferLabels' 'file' "location of FreeSurferAllLut.txt"
 opts_AddMandatory '--refmyelinmaps' 'ReferenceMyelinMaps' 'file' "high-resolution group myelin map to use for bias correction"
+opts_AddMandatory '--msm-all-templates' 'MSMAllTemplates' 'path' "path to directory containing MSM All template files, e.g. 'YourFolder/global/templates/MSMAll'"
```

Comment:
This is an odd variable name for PostFreeSurfer that precedes MSMAll by quite a few lines.  I am not a huge fan of tying the myelin target file to a specific folder either.  I would have preferred a full path and a single argument.

Reply:
I changed the myelin target as a single argument with an absolute path wherever the MyelinMap_BC is involved.